readme.md

```diff
@@ -110,4 +111,4 @@ Once questions are stored the next thing to do is fetch the answers. Some answer
 - Yahoo only shows downvotes when you sign in
 - When fetching answers, I found a few duplicates in the question ids... so for now I modified mysql errors to write to a file instead of exiting as to not interrupt the program. Question id's are: 20131020103951AAJqaTE, 20131016175727AAa1l43, 20131015174409AAImTMz
 - chosen_answer_timestamp and chosen_answer_award_timestamp did not get filled
-- It seems like after so many questions are fetched, you get blank results back
\ No newline at end of file
+- It seems like after so many questions are fetched, you get blank results back
```

Comment:
Can you revert the changes to this line? I'm not sure why this says it's changed.